### PR TITLE
Update GitHub Community on Discussions details

### PR DESCRIPTION
**Pull request by Akash1134**
### Why:

The description for GitHub Community on Discussions is [outdated](https://docs.github.com/en/get-started/learning-about-github/faq-about-changes-to-githubs-plans#what-is-github-community-support) in the `FAQ about changes to GitHub’s plans` doc.

We want to update it and make it consistent with our Community [ReadME](https://github.com/community/.github/blob/main/profile/README.md). 

Closes ISSUE

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Changes Required:

- Title: From "What is GitHub Community Support" to " What is GitHub Community on Discussions"
- Description: Replace with the following:
"GitHub Community is built to support all GitHub users on their educational journey, via Discussions. It is a resource hub, learning portal, and inspiration station, all in one. Regardless of how big or small your challenge is, all resources and information will be accessible in a true open-source fashion.”

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @Akash1134 - Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/get-started/learning-about-github/faq-about-changes-to-githubs-plans.md [hunk 1]
```diff
@@ -75,11 +75,11 @@ Now that {% data variables.product.prodname_team %} has no minimum seat requirem
 
 ## Is anything changing for teams on the GitHub Team for Open Source plan?
 
-If your organization account currently uses the GitHub Team for Open Source plan, you’ve been automatically moved to GitHub Free, which includes private repositories with unlimited collaborators, 2,000 Actions minutes for private repositories, and GitHub Community Support. Actions minutes and Packages storage are still free for your public repositories.
+If your organization account currently uses the GitHub Team for Open Source plan, you’ve been automatically moved to GitHub Free, which includes private repositories with unlimited collaborators, 2,000 Actions minutes for private repositories, and GitHub on Discussions. Actions minutes and Packages storage are still free for your public repositories.
```
Comment (hubwriter) [inline content/get-started/learning-about-github/faq-about-changes-to-githubs-plans.md:78]: I think there's a missing "Community" after "GitHub". I'll add this.
```suggestion
If your organization account currently uses the GitHub Team for Open Source plan, you’ve been automatically moved to GitHub Free, which includes private repositories with unlimited collaborators, 2,000 Actions minutes for private repositories, and GitHub Community on Discussions. Actions minutes and Packages storage are still free for your public repositories.
```

**Review by hubwriter — DISMISSED** [review]
@Akash1134 - Many thanks for raising this PR. 👍 

I've changed "2,000 Actions minutes for private repositories, and GitHub on Discussions" to "2,000 Actions minutes for private repositories, and GitHub Community on Discussions" - because I think the word "Community" was missing there.

I'll get this merged.

**Review by cmwilson21 — APPROVED** [review]

**Comment by cmwilson21** [comment]
Closing and reopening this PR to jump start status checks 👍

**Review by Giangpro89 — COMMENTED** [review]
Tuyệt vời

**Pull request merged**